ovn-tester/ovn_tester.py

```diff
@@ -125,11 +121,14 @@ def read_config(config):
         physical_net=cluster_args.get("physical_net", "providernet"),
     )
 
-    bringup_args = config.get("base_cluster_bringup", dict())
-    bringup_cfg = ClusterBringupCfg(
-        n_pods_per_node=bringup_args.get("n_pods_per_node", 10)
-    )
-    return global_cfg, cluster_cfg, brex_cfg, bringup_cfg
+    # Apart from the 'name' attribute, which is mandatory to load the CMS
+    # plugin, we do not attempt to parse CMS specific configuration.
+    cms_args = config.get("cms", dict())
```

Comment:
That is an intriguing suggestion. I guess that would work because the tests are always executed in order.

The thing that made me avoid `GlobalCfg` was that I needed to update `ovn-fake-multinode-utils/translate_yaml.py` for it to work, which did feel a bit wrong, hence the new top level node.

But I guess there is nothing wrong with using the global config for this after all.